wp-content/themes/borderzine/partials/widget-content.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+// The top term
+if ( isset( $instance['show_top_term'] ) && $instance['show_top_term'] == 1 && largo_has_categories_or_tags() ) {
+	largo_maybe_top_term();
+}
+
+
+// the thumbnail image (if we're using one)
+if ($thumb == 'small') {
+	$img_location = ! empty( $instance['image_align'] ) ? $instance['image_align'] : 'left';
+	$img_attr = array( 'class' => $img_location . '-align' );
+	$img_attr['class'] .= " attachment-small";
+	?>
+		<a href="<?php echo get_permalink(); ?>"><?php echo get_the_post_thumbnail( get_the_ID(), '60x60', $img_attr); ?></a>
+	<?php
+} elseif ($thumb == 'medium') {
+	$img_location = ! empty( $instance['image_align'] ) ? $instance['image_align'] : 'left';
+	$img_attr = array('class' => $img_location . '-align');
+	$img_attr['class'] .= " attachment-thumbnail";
+	?>
+		<a href="<?php echo get_permalink(); ?>"><?php echo get_the_post_thumbnail( get_the_ID(), 'post-thumbnail', $img_attr); ?></a>
+	<?php
+} elseif ($thumb == 'large') {
+	$img_attr = array();
+	$img_attr['class'] = " attachment-large";
+	?>
+		<a href="<?php echo get_permalink(); ?>"><?php echo get_the_post_thumbnail( get_the_ID(), 'large', $img_attr); ?></a>
+	<?php
+} elseif( $thumb == 'rect_thumb_half' ){
+    $img_attr = array();
+    $img_attr = 'attachment-rect-thumb-half';
+    ?>
+        <a href="<?php echo get_permalink(); ?>"><?php echo get_the_post_thumbnail( get_the_ID(), 'rect_thumb_half', $img_attr); ?></a>
```

Comment:
The image markup output here, with added whitespace, is:

```html
<img
	src="https://borderzine.test/wp-content/uploads/2019/05/Little-Habana-Cuban-restaurant-Juarez.jpg"
	class="attachment-rect_thumb_half size-rect_thumb_half wp-post-image"
	alt=""
	attachment-rect-thumb-half=""
	srcset="https://borderzine.test/wp-content/uploads/2019/05/Little-Habana-Cuban-restaurant-Juarez.jpg 938w,
		https://borderzine.test/wp-content/uploads/2019/05/Little-Habana-Cuban-restaurant-Juarez-150x81.jpg 150w,
		https://borderzine.test/wp-content/uploads/2019/05/Little-Habana-Cuban-restaurant-Juarez-600x325.jpg 600w,
		https://borderzine.test/wp-content/uploads/2019/05/Little-Habana-Cuban-restaurant-Juarez-610x330.jpg 610w"
		sizes="(max-width: 400px) 100vw, 400px" width="400" height="217"
>
```

The full-size image is 938x508px; the image that's getting loaded on my device is the uncropped full-size image:

![Screen Shot 2019-11-21 at 01 15 09](https://user-images.githubusercontent.com/1754187/69312241-61e0ad80-0bfc-11ea-862d-099cf9580c0c.png)

But reuploading the image gets a properly-sized image used as the thumbnail:

![Screen Shot 2019-11-21 at 01 19 21](https://user-images.githubusercontent.com/1754187/69312446-f814d380-0bfc-11ea-8b39-a3e485287116.png)

In conclusion:
- they need to use big-enough images for the listing images moving forwards, where "big enough" is at least 400x300
- we may need to run the regnerate thumbnails plugin

Reply:
Adding a note to https://github.com/INN/umbrella-borderzine/issues/43 about running the regenerate thumbnails plugin.